### PR TITLE
Breakout AI into individual projects like LUIS

### PR DESCRIPTION
**Pull request by garypretty**
In order to improve consistency and in answer discussion in #324, I have done the following;

* Split out QnA and Translation into their own Projects, to match the pattern used with LUIS, within a Microsoft.Bot.Builder.AI namespace (also moved LUIS into this namespace)
* Introduced an AI solution folder to allow for more projects to be added in the future
* Updated samples and unit test project references (all unit tests pass and samples build)

@cleemullins This work is also based on a conversation we had previously where you suggested you were open to splitting AI out further.  I think it makes sense so that individual packages can be released and updated as and when - potentially even owned by different teams etc.

### CODE REVIEW

**Comment by tomlm** [comment]
> 	<PackageId>Microsoft.Bot.Builder.LUIS</PackageId>

Shouldn't the package name change to Microsoft.Bot.Builder.Ai.Luis?

---
Refers to: libraries/Microsoft.Bot.Builder.LUIS/Microsoft.Bot.Builder.Ai.LUIS.csproj:17 in ca67c9c. [](commit_id = ca67c9c7f0c9628a36c4a0e12ba95b2339b8c285, deletion_comment = False)

File: libraries/Microsoft.Bot.Builder.QnA/Microsoft.Bot.Builder.Ai.QnA.csproj [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
   <PropertyGroup>
 	  <TargetFramework>netstandard2.0</TargetFramework>
-	  <PackageId>Microsoft.Bot.Builder.Ai</PackageId>
-    <Description>AI classes for using Cognitive Services on the Microsoft Bot Builder SDK</Description>
+	  <PackageId>Microsoft.Bot.Builder.QnA</PackageId>
```
Comment (tomlm) [inline libraries/Microsoft.Bot.Builder.QnA/Microsoft.Bot.Builder.Ai.QnA.csproj]: >QnA [](start = 36, length = 3)

Microsoft.Bot.Builder.Ai.QnA?

**Comment by garypretty** [comment]
Thanks. I'll check these tomorrow and amend. Looks like I'll need to do a merge with latest too with a couple of large pull requests going in potentially. Thanks for the review!

**Comment by garypretty** [comment]
@cleemullins I am going to wait on updating this PR until you have merged #340

**Comment by garypretty** [comment]
@tomlm @cleemullins Actually, looking at it, with the QnA sample getting updated and my PR moving projects / namespaces around, I am going to close this PR and re-do the work based on Chris' changes in #340. I will re-open a PR later today which should have no conflicts 👍

**Comment by garypretty** [comment]
@cleemullins @tomlm By way of an update - I have a new PR ready to raise for this based on Chris' branch #340 for as soon as that is merged.

**Pull request closed without merging**